### PR TITLE
Rewrite IT case to remove Vertx web client

### LINKED ISSUE #150: Check deps
Check if we really need `vertx-web-client` dependency and update it if needed (#125 automated update failed)

### PR DESCRIPTION
**Pull request by genryxy**
Closes #150 
`HelmSliceIT` was refactored to remove `vertx-web-client` dependency as the same could be done using

### CODE REVIEW

File: src/test/java/com/artipie/helm/HelmSliceIT.java [hunk 1]
```diff
@@ -24,143 +24,60 @@
...
-                    .and(new IsMapContaining<>(new IsEqual<>("created"), new IsAnything<>()))
-            );
-        } finally {
-            web.close();
-            server.close();
-            vertx.close();
-        }
-    }
-
+final class HelmSliceIT {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/helm/HelmSliceIT.java:51]: @genryxy hm, now this test is not integration test as it does not use/communicate with any external systems...

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy please, consider my comment

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected. Moved test case from `HelmITCase` to `HelmCompatibilityITCase` and renamed `HelmCompatibilityITCase` to `HelmITCase`

**Comment by olenagerasimova** [comment]
@genryxy thanks

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**